src/inject/single_injection.py

```diff
@@ -26,17 +25,17 @@ def __init__(
         patch_file: Path,
         patch_line_no: int,
     ) -> None:
-        self.file_name = file_name         # file to inject into
-        self.file_location = file_location # location to inject at
+        self.file_name = file_name          # file to inject into
```

Comment:
Consider putting comments on the line above rather than to the side, so that they don't bloat the line length.

Reply:
Sounds good!